co-log-core/src/Colog/Core/IO.hs

```diff
@@ -150,3 +151,11 @@ foo
 liftLogIO :: MonadIO m => LogAction IO msg -> LogAction m msg
 liftLogIO (LogAction action) = LogAction (liftIO . action)
 {-# INLINE liftLogIO #-}
```

Comment:
As a side note I noticed that `liftLogIO` lacks the `SPECIALIZE` pragma. Is it worth adding in this PR as a separate commit?

Reply:
Though probably it's not really useful because if `m` is `IO` this function is `id` :thinking: